### DISCUSSION TITLE
Is it allowed to paste my own answer in a kata that has the same logic as another kata?

### DISCUSSION THREAD

**Discussion by mahdizarei0614** [codewars/codewars.com#3007] [Q&A] [answered]
Hi, I solved [Insane Coloured Triangles](https://www.codewars.com/kata/5a331ea7ee1aae8f24000175) kata, then I encountered [Ludicrous Coloured Triangles](https://www.codewars.com/kata/5b84d6d6b2f82f34d00000d7) kata, which I found similar and I copy and pasted my answer in the second kata and in the first attempt it was done and then I submitted my answer.

Is this act forbidden or what?

**Comment by Blind4Basics** [comment, marked as answer]
it's not forbidden, but it's a problem => mods have to hide the solution in the easier problem.
done.

**Reply by mahdizarei0614** [reply]
they are both 2kyu

**Reply by Blind4Basics** [reply]
and? the ludicrous one is still harder.

**Reply by jarede-dev** [reply]
"mods have to hide the solution in the easier problem"

why?

**Reply by Blind4Basics** [reply]
otherwise you see loads of users who just copy/paste the solution they found in the easy kata in the harder one, making the harder version pointless.
